internal/build/get_version.sh

```
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+set -eu
+
+xy_ver() {
+	local ver="${1:-}"
```

Shellcheck

[warning] In POSIX sh, 'local' is undefined.

(SC3043)